osmaxx/core/templates/pages/about_us.html

```diff
@@ -8,6 +8,12 @@
         OSMaxx (short for "OpenStreetMap arbitrary excerpt export") cuts out OpenStreetMap data, processes it to
         geodata, converts it to typical GIS fileformats and offers those for download.
     </p>
+    <h3>Documentation</h3>
+    <h4>Schema</h4>
+    <p>
+        You can find the online documentation of the schema on GitHub:
+        <a href="https://github.com/geometalab/osmaxx-docs/blob/master/documentation.md">Export Schema Documentation</a>.
```

Comment:
very good, I actually thought this ought to be on the code repo, but it is already :-)
removing would actually be the right thing to do ;-)